pkg/errors/errors.go

```diff
@@ -232,3 +234,24 @@ func NewUserNotAllowedError(image string, onbuild bool) error {
 		Suggestion: fmt.Sprintf("modify image %q to use a numeric user within the allowed range or build without the --allowed-uids flag", image),
 	}
 }
+
+// NewMissingGitBinaryError returns a new error that indicates that we couldn't
+// find a Git binary in the path.
+func NewMissingGitBinaryError() error {
```

Comment:
@oatmealraisin Ryan, this method looks unused. Should we use it somewhere or just remove?